content/add-a-printer-pop.md

```diff
@@ -57,7 +57,7 @@ Brother provides a driver installation tool for Linux users. Install the appropr
 
 6. Press <kbd>Super</kbd> + <kbd>T</kbd> to launch the Terminal application.
 
-7. Change directory to where you downloaded the driver, then unzip it. This is usually the downloads directory.
+7. Change directory to where you downloaded the driver, then unzip it. This is usually the downloads directory. Note that those stars are placeholders for the version number of the installer, feel free to put the version number in place of each of those stars.
```

Comment:
Although it isn't entirely necessary. Bash will wildcard them and it should work anyway. Heck, it could probably even be shortened to `gunzip linux-brprinter-installer-*.gz`

Reply:
Good idea. Changed the command to your suggestion.